solr/build.gradle

```diff
@@ -0,0 +1,41 @@
+plugins {
+    id "maven"
+    id "maven-publish"
+    id "com.palantir.docker"
+}
+
+group = "com.connexta.search"
+version = "0.1.0-SNAPSHOT"
```

Comment:
Yes because I version the solr config zip artifact. I'm questioning if this artifact will even be needed though! I'm just going to actually remove deploying the artifact for now. My original motivation was to pull it to configure an in memory solr for itests rather than starting the solr docker image. But not I'm thinking it's more valuable to test the docker image directly since that is what is actually used in production.